docs/configuring.md

```diff
@@ -432,6 +420,43 @@ settings that are defined in more than one document:
         - "pyproject.toml"
     ```
 
+The structure of includes is quite restrictive: they must come at the top of the yaml
+file, and end with `---` on a new line.
+
+!!! example
+
+    This is an _invalid_ copier.yml file:
+
+    ```yaml title="copier-invalid.yml"
+    version:
+        type: str
+        help: What is the version of your Python project?
+    
+    !include copier-other.yml
+    ```
+
+    This is invalid twice over: we haven't put the `!include` directive at the top
```

Comment:
How about just demonstrating and immediately annotating the *correct* syntax:
```
!include copier-other.yml
---
```
Please note that the `!include` directive **MUST** be the **first line** in the file.

The line after the directive must contain **only** three dashes and a newline character as shown above.

*Anything else is invalid and will not work.*
```

Reply:
> Please note that the `!include` directive **MUST** be the **first line** in the file.
> 
> The line after the directive must contain **only** three dashes and a newline character as shown above.
> 
> _Anything else is invalid and will not work._

That's not correct. The `!include` directive can appear anywhere in `copier.yml` as long as it's placed in a separate YAML document. For instance:

* `copier.yml`;
    ```yaml
    q1: hello
    ---
    !include q2.yml
    ```
* `q2.yml`:
    ```yaml
    q2: world
    ```

Reply:
yes